### ISSUE TITLE
small memleak on RAI_ScriptCreateTorch

### ISSUE THREAD

**Issue by filipecosta90** [RedisAI/RedisAI#281]
```
==26251== 4 bytes in 1 blocks are definitely lost in loss record 35 of 132,287
==26251==    at 0x4C2FB0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==26251==    by 0x144D38: zmalloc (zmalloc.c:99)
==26251==    by 0x144F83: zstrdup (zmalloc.c:210)
==26251==    by 0x1C7006: RM_Strdup (module.c:352)
==26251==    by 0x6A7AA2B: RAI_ScriptCreateTorch (torch.c:154)
==26251==    by 0x67EC4CE: RAI_ScriptCreate (script.c:92)
==26251==    by 0x67DBE1F: RedisAI_ScriptSet_RedisCommand (redisai.c:1353)
==26251==    by 0x1C76BE: RedisModuleCommandDispatcher (module.c:575)
==26251==    by 0x13D315: call (server.c:2468)
==26251==    by 0x13DFCA: processCommand (server.c:2763)
==26251==    by 0x14F086: processInputBuffer (networking.c:1470)
==26251==    by 0x14F1DB: processInputBufferAndReplicate (networking.c:1505)
```

**Comment by lantiga** [comment]
Fixed in #289

**Comment by filipecosta90** [comment]
fixed :+1:
